drf_jsonapi/mixins.py

```diff
@@ -43,8 +59,11 @@ def list(self, request, *args, **kwargs):
             many=True,
             only_fields=request.fields,
             include=request.include,
-            page_size=request.GET.get('page[size]', settings.DEFAULT_PAGE_SIZE),
-            context={'request': request}
+            page_size=request.GET.get(
+                "page[size]",
+                getattr(settings, "DEFAULT_PAGE_SIZE", defaults.DEFAULT_PAGE_SIZE),
+            ),
+            context={"request": request},
```

Comment:
@chrisbrantley would you consider adding the view itself to the context object? Or perhaps just the view.action property? Recently I encountered a case where that information would have been very helpful. If you're agreeable to it I'd like to see it added to all of the mixins.

Reply:
That’s interesting. I don’t suppose it would hurt to pass that in. Can you help me understand what the use case would be?